myuw/static/vue/components/_templates/boilerplate/boilerplate.vue

```diff
@@ -400,86 +400,47 @@ export default {
     ...mapMutations(['addVarToState']),
     generateUserProperties() {
       const properties = {};
-      const affiliationEntries = Object.entries(this.affiliations)
-        .filter((pair) => pair[1]);
+      properties['affiliation'] = (
+        this.affiliations['applicant'] ? 'Applicant' :
+        this.affiliations['faculty'] ? 'Faculty' :
+        this.affiliations['staff_employee'] ? 'Staff' :
+        this.affiliations['student'] ? 'Student' :
+        this.affiliations['alumni'] ? 'Alumni' : null
+      );
+
+      properties['class_level'] = this.titleCaseWord(this.affiliations['class_level']);
 
-      this.populateProperty(
-        properties,
-        'affiliation',
-        {
-          'alumni': 'Alumni',
-          'applicant': 'Applicant',
-          'clinician': 'Clinician',
-          'faculty': 'Faculty',
-          'grad': 'Grad stud',
-          'grad_c2': 'GradC2',
-          'instructor': 'Instructor',
-          'intl_stud': 'Intl stud',
-          'past_stud': 'Former stud',
-          'pce': 'PCE',
-          'retiree': 'Retiree',
-          'staff_employee': 'Staff',
-          'stud_employee': 'Stud employee',
-          'student': 'Student',
-          'undergrad': 'Undergrad stud',
-          'undergrad_c2': 'UndergradC2',
-        },
-        affiliationEntries,
+      properties['continuum_college'] = (
+        this.affiliations['grad_c2'] ? 'GradC2' :
+        this.affiliations['undergrad_c2'] ? 'UndergradC2' :
+        this.affiliations['pce'] ? 'PCE' : null
       );
-      this.populateProperty(
-        properties,
-        'student_campus',
-        {
-          'seattle': 'Seattle',
-          'bothell': 'Bothell',
-          'tacoma': 'Tacoma',
-        },
-        affiliationEntries,
-        true,
+
+      properties['emp_position'] = (
+        this.affiliations['instructor'] ? 'Instructor' :
+        this.affiliations['stud_employee'] ? 'Stud employee' :
+        this.affiliations['clinician'] ? 'Clinician' :
+        this.affiliations['retiree'] ? 'Retiree' : null
       );
-      this.populateProperty(
-        properties,
-        'employment_campus',
-        {
-          'official_seattle': 'Seattle',
-          'official_bothell': 'Bothell',
-          'official_tacoma': 'Tacoma',
-        },
-        affiliationEntries,
-        true,
+
+      properties['student_prop'] = (
+        this.affiliations['intl_stud'] ? 'International' :
+        this.affiliations['past_stud'] ? 'Former' : null
```

Comment:
`Former` is not documented in https://docs.google.com/document/d/1p8SxGS_ha4MtiOTDapXPa7Nnu8lRjwT8zAuoPHnIZx0/edit?usp=sharing